packages/assets/package.json

```diff
@@ -5,10 +5,6 @@
   "main": "lib/index",
   "license": "MIT",
   "author": "Andy Wermke <andy@dev.next-step-software.com>",
-  "scripts": {
-    "test": "ava",
-    "prepublishOnly": "yarn test"
```

Comment:
Nope, manual only for now. I would like to tickle it in another PR.

- Publishing single packages leads to version inconsistencies and bugs. For instance the `uglify` block was publish separately as 1.0.0 and later as 1.1.0, when other blocks used to be v1.0.0-beta. Then with the 1.0 release the `uglify` was republished and its version 1.0.0 was tagged as latest. What that means is if you run `yarn add @webpack-blocks/uglify` now, you will get 1.0.0 instead of 1.1.0. (I am sorry I haven't reported it before 😅)
- Some blocks have no unit but instead have related e2e tests. I believe these are more important to have and we can focus on writing more of them.

Reply:
Oh okay. Then we should just create a small doc to refer to when publishing. Like that we only publish "whole monorepo", not standalone.